lib/wcc/contentful/store/cdn_adapter.rb

```diff
@@ -9,7 +9,9 @@ class CDNAdapter
 
     def initialize(client)
       super()
-      if client.client_type == 'preview'
+      if client.nil?
```

Comment:
When would client be nil?  Shouldn't that situation cause an argument error?

Also I still don't think we need to have both a `@client` and a `@preview_client`.  They both implement the same interface, correct?